triton-lang/triton
# [AMD] Use warp shuffle for MFMA to Dot operand layout conversion (FP8)

Adding a case for MFMA->Dot (FP8) layout conversion that avoids using shared memory, to speed up FP8 attention kernels. 
Test:
lit test
ctest -j32


  [rules](https://cbea.ms/git-commit/#why-not-how).


- Select one of the following.
    - `/test` for `lit` tests
    - `/unittest` for C++ tests
    - `/python/test` for end-to-end tests

- Select one of the following.
    including the "tests should be minimal" section. (Usually running Python code
    and using the instructions it generates is not minimal.)
